docs/core/compendium_de.md

```diff
@@ -0,0 +1,1177 @@
+   {
+     "Name": "service-account-create",
+     "Description": "Creates a new service account",
+     "Command": "/create-sa.sh"
+   },
+   {
+     "Name": "service-account-remove",
+     "Description": "Removes a service account",
+     "Command": "/remove-sa.sh"
+   }
+ ],
+```
+
+Benutzerdefinierte Befehle: Darüber hinaus kann ein Dogu Befehle angeben, die nicht automatisch im Lebenszyklus des
+Dogus ausgeführt werden (z.B. Upgrade). Zum Beispiel kann ein Dogu wie Redmine einen Befehl zum Löschen oder
+Installieren eines Plugins angeben. Zur Laufzeit kann ein Administrator diesen Befehl mit der cesapp wie folgt aufrufen:
+
+```
+cesapp command redmine plugin-install scrum-plugin
+```
+
+Beispiel:
+
+```
+"ExposedCommands": [
+   {
+     "Name": "plugin-install",
+     "Description": "Installs a plugin",
+     "Command": "/installPlugin.sh"
+   }
+ ],
+```
+
+### Volumes
+
+Volumes enthält eine Liste von [Volume](#type-volume), die [OCI container volumes](https://opencontainers.org/)
+definiert. Dieses Feld ist optional.
+
+Volumes werden während der Erstellung oder des Upgrades eines Dogus erstellt. Volumes bieten eine Leistungssteigerung im
+Vergleich zum Container-Speicher.
+
+Beispiele:
+
+```
+[ { "Name": "temp", "Path":"/tmp", "Owner": "1000", "Group": "1000", "NeedsBackup": false} ]
+```
+
+### HealthCheck
+
+HealthCheck definiert eine einzige Möglichkeit, den Zustand des Dogus auf Beobachtbarkeit zu prüfen. Dieses Feld ist
+optional.
+
+Veraltet: verwenden Sie stattdessen [HealthChecks](#healthchecks).
+
+### HealthChecks
+
+HealthChecks definieren eine Liste von [HealthCheck](#type-healthcheck). Sie bieten mehrere Möglichkeiten, den Zustand
+des Dogus auf Beobachtbarkeit zu prüfen. Dieses Feld ist optional.
+
+HealthChecks werden in verschiedenen Anwendungsfällen verwendet:
+
+- um die `dogu is starting` Seite anzuzeigen, bis das Dogu beim Start healthy ist
+- zur Überwachung über `cesapp healthy <dogu-name>`
+- für die Überwachung über das Admin-Dogu
+- um zu vermeiden, dass ein unhealthy Dogu gesichert wird
+
+Es gibt verschiedene Arten von HealthChecks:
+
+- state, über den `/state/<dogu>` Registry-Schlüssel, der von ces-setup gesetzt wird
+- tcp, um auf einen offenen Port zu prüfen
+- http, um einen Statuscode einer http-Antwort zu prüfen
+
+Sie müssen ohne Authentifizierung ausführbar sein.
+
+Beispiel:
+
+```
+"HealthChecks": [
+  {"Type": "state"}
+  {"Type": "tcp",  "Port": 8080},
+  {"Type": "http", "Port": 8080, "Path": "/my/health/path"},
+]
+```
+
+### ServiceAccounts
+
+ServiceAccounts enthält eine Liste von [ServiceAccount](#type-serviceaccount). Dieses Feld ist optional.
+
+Ein Service-Account schützt sensible Daten, die für ein anderes Dogus verwendet werden. Service-Account-Konsumenten
+werden von Service-Account-Produzenten unterschieden. Um ein Servicekonto zu erzeugen, muss ein Dogu die Skripte
+service-account-create und service-account-delete in [core.ExposedCommands](#exposedcommands) implementieren.
```

Comment:
weird text structure / too passive language
I am torn between a more common, though passive sentence like
```
Man unterscheidet Service-Account-Konsumenten von Service-Account-Produzenten.implementieren.
```
and a more active voice which reads a bit more awkward:
```
Service-Account-Konsumenten unterscheiden sich hierbei von Service-Account-Produzenten.
```